packages/ingest/aws.js

```diff
@@ -49,11 +49,13 @@ function getEndpoint(local = false, port = 8000) {
  * @returns {string} Step Function Execution Arn
  */
 function getExecutionArn(stateMachineArn, executionName) {
-  if (stateMachineArn && executionName) {
-    const sfArn = stateMachineArn.replace('stateMachine', 'execution');
-    return `${sfArn}:${executionName}`;
-  }
-  return null;
+  deprecate(
+    '@cumulus/ingest/aws/getExecutionArn()',
+    '1.13.0',
```

Comment:
I wasn't sure what the right version number is to use for the deprecation notice?

Reply:
1.13.4 is probably the better one now, since this change won't be in that version.